packages/@lwc/compiler/package.json

```
@@ -23,7 +23,7 @@
         "@lwc/template-compiler": "1.3.6",
         "@rollup/plugin-replace": "^2.3.1",
         "babel-preset-compat": "^0.22.0",
-        "rollup": "^1.32.0",
+        "rollup": "^1.32.1",
```

The convention we are trying adopt is `~ for dep and ^ for dev dep`
```suggestion
        "rollup": "~1.32.1",
```

---

I would rather make all those changes at once and keep it consistent for now.